HelpAssistHer/help-assist-her
# Style About Page

## Description
Add the basic styling for the About page. This doesn't include any of the team images or bios.

Also note that the copy is not finalized, but we'll tweak at a later time.

<img width="403" alt="Screen Shot 2019-04-10 at 8 29 54 PM" src="https://user-images.githubusercontent.com/6378435/55928976-82a94500-5bcf-11e9-9096-d9d0f8c99f8f.png">

## Test Plan
1. Go to http://localhost:4000/about
1. Verify the About page matches the designs
1. Make sure the page looks good for both mobile and desktop views
